Forms/Form.php

```diff
@@ -1377,14 +1370,17 @@ public function getLegend() {
 	}
 
 	/**
-	 * Returns last result of validation, or null if not validated.
-	 * Note: Does not persist between requests in session.
-	 * @todo Maybe it should?
+	 * Returns boolean if there are any errors set for this form, or null if not validated.
 	 *
 	 * @return bool|null
 	 */
 	public function isValid() {
```

Comment:
Hm, how about FormStatus, with a return value of one of the following consts:

 - UNSUBMITTED
 - SUBMITTED_VALID
 - SUBMITTED_INVALID